test/sidekick-test.js

```diff
@@ -271,6 +283,77 @@ export class SidekickTest {
     return this;
   }
 
+  /**
+   * Mocks an admin DOM to test bulk operations
+   * @param {string} [contentSource] The content source: "sharepoint" (default) or "gdrive"
+   * @param {import('@Types').BulkSelection} [resources] The resources
+   * @param {string} [viewType] The view type: "list" (default) or "grid"
+   * @returns {SidekickTest}
+   */
+  mockAdminDOM(
+    contentSource,
+    viewType,
+    resources,
+  ) {
+    if (!contentSource) {
+      contentSource = HelixMockContentSources.SHAREPOINT;
+    }
+    if (!resources) {
+      resources = contentSource === HelixMockContentSources.SHAREPOINT
+        ? DEFAULT_SHAREPOINT_BULK_SELECTION
+        : DEFAULT_GDRIVE_BULK_SELECTION;
+    }
+
+    this.mockEditorAdminEnvironment(
+      EditorMockEnvironments.ADMIN,
+      HelixMockContentType.ADMIN,
+      contentSource,
+    );
+
+    let root;
+    if (contentSource === HelixMockContentSources.SHAREPOINT) {
+      // mock sharepoint icon requests
+      fetchMock.get('glob:http://localhost:2000/icons/**', {
+        status: 200,
+      }, { overwriteRoutes: true });
```

Comment:
@dylandepass I tried a number of different ways here, including glob and regexp, but to no avail: the test output always shows:
```
🚧 404 network requests:
    - icons/docx.svg
    - icons/xlsx.svg
    - icons/pdf.svg
    - icons/image.svg
    - icons/video.svg
    - icons/svg.svg
    - icons/unknown.svg
    - icons/foldericons/folder.svg
```
Any ideas how to mock these?